bigquery/google/cloud/bigquery/client.py

```diff
@@ -1187,7 +1189,9 @@ def extract_table(
         return extract_job
 
     def query(
-            self, query, job_config=None, job_id=None, job_id_prefix=None,
+            self, query,
+            job_config=None, override_job_config=False,
```

Comment:
Can we rename `override_job_config` to `merge_job_config`, and invert its meaning?  I think that would be clearer.  It needs to be added to the method docstring below.

Reply:
:+1: in regards to adding it to the docstring.

Just to clarify about `override_job_config`, I'll flip the argument default to also be `True` correct? It seems like the default behavior should be to use the default you've previously passed, and it would seem annoying to clarify in every `query` call that that's what you want.

Reply:
Tests need to accommodate this renaming, too.